be/src/olap/memtable_flush_executor.cpp

```diff
@@ -95,17 +95,14 @@ MemTableFlushExecutor::~MemTableFlushExecutor() {
         queue->shutdown();
     }
 
-    // shutdown thread pool
-    _flush_pool->shutdown();
-    _flush_pool->join();
+    // The threadpool will be shutdown and joined in dtor
+    delete _flush_pool;
```

Comment:
I prefer the former method because it is clear. The way you changed is not clear and has the risk that someone may change the thread_pool's imeplmentation later

Reply:
Done.